### PR TITLE
Only migrate active and disable connection

### PR DESCRIPTION
**Pull request by benmoriceau**
## What
Filter out the deleted workflows.

Change a log to the right level.

### CODE REVIEW

File: airbyte-scheduler/app/src/main/java/io/airbyte/scheduler/app/SchedulerApp.java [hunk 1]
```diff
@@ -234,13 +232,13 @@ public static void main(final String[] args) throws IOException, InterruptedExce
         configs.getDatabaseUser(),
         configs.getDatabasePassword(),
         configs.getDatabaseUrl())
-            .getInitialized();
+        .getInitialized();
```
Comment (lmossman) [inline airbyte-scheduler/app/src/main/java/io/airbyte/scheduler/app/SchedulerApp.java:235]: Probably need to run `gw format`

File: airbyte-server/src/main/java/io/airbyte/server/ServerApp.java [hunk 2]
```diff
@@ -235,14 +236,15 @@ private static void migrateExistingConnection(final ConfigRepository configRepos
       throws JsonValidationException, ConfigNotFoundException, IOException {
     LOGGER.info("Start migration to the new scheduler...");
     final Set<UUID> connectionIds =
-        configRepository.listStandardSyncs().stream().map(standardSync -> standardSync.getConnectionId()).collect(Collectors.toSet());
+        configRepository.listStandardSyncs().stream()
+            .filter(standardSync -> standardSync.getStatus() == Status.ACTIVE || standardSync.getStatus() == Status.INACTIVE)
```
Comment (lmossman) [inline airbyte-server/src/main/java/io/airbyte/server/ServerApp.java:240]: What is the reasoning behind filtering down to only ACTIVE and INACTIVE?

File: airbyte-scheduler/app/src/main/java/io/airbyte/scheduler/app/SchedulerApp.java (hunk 1 above)
Comment (benmoriceau) [reply airbyte-scheduler/app/src/main/java/io/airbyte/scheduler/app/SchedulerApp.java]: Done

File: airbyte-server/src/main/java/io/airbyte/server/ServerApp.java (hunk 2 above)
Comment (benmoriceau) [reply airbyte-server/src/main/java/io/airbyte/server/ServerApp.java:240]: ACTIVE = enabled workflow

Comment (lmossman) [reply airbyte-server/src/main/java/io/airbyte/server/ServerApp.java:240]: I think that makes sense to me. Though we should probably get a confirmation from @jrhizor to be sure

**Review by lmossman — APPROVED** [review]

**Pull request merged**